examples/training/1-Intro to Bricks.ipynb

```diff
@@ -359,7 +359,7 @@
     {
      "data": {
       "text/plain": [
-       "'[1]\\xa0Geolocated combat footage has confirmed Russian gains in the Dvorichne area northwest of Svatove.'"
+       "'Search form'"
```

Comment:
Those look like the banners at the top of the page. Are the writeup portions not getting flagged as `NarrativeText` now?

<img width="627" alt="image" src="https://user-images.githubusercontent.com/1635179/221680730-1dac5a90-99a5-4396-acd7-624014d5753c.png">

Reply:
No NarrativeText is found among the elements.

Reply:
<img width="1301" alt="image" src="https://user-images.githubusercontent.com/3939469/221692062-8b36b0fa-a85a-4324-96d7-478873932cde.png">